src/styles/blocks/star.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  &--1 &__star:nth-child(-n + 1) {
+    background-image: $star-img;
+  }
```

Comment:
You are using `nth-child` here and `nth-of-type` in lines 21, 25, and 29. Stick to one method for consistency and to avoid confusion. The [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) recommend consistency in selectors.